vertx-sql-client/src/main/java/io/vertx/sqlclient/RowSet.java

```diff
@@ -20,7 +20,7 @@
 import io.vertx.codegen.annotations.VertxGen;
 
 /**
- * A set of rows.
+ * The execution resultSet which in the form of {@link Row rows}.
```

Comment:
This is incorrect, this is a generic type. Instead it should be 

"The execution result of the row set of a query provided as {@code <R>}, commonly used as a {@code RowSet<Row>}. Using a collector query might might provide a different {@code <R>} result"

Reply:
Currently there's only `RowSet<Row>` in the public APIs and it does not provide any public ways to construct a `RowSet<R>`, is there any use case to represent the result as a row set of generic objects? And the collector query gives the result as `SqlResult<R>` instead of `RowSet<R>`.